### ISSUE TITLE
Updating the documentation for Language Options: language:xx

### ISSUE THREAD

**Issue by neffmartin** [totaldebug/atomic-calendar-revive#769]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Locale Support just tells, that there are language option but not how to configure them.

### Expected behaviour

add Locale Support the option you have to add to the config file:
language: xx   <- xx = Locale

### Possible Solutions

add Locale Support the option you have to add to the config file:
language: xx   <- xx = Locale

### Context / Reason

couldn't find the solution for a long time

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by marksie1988** [comment]
Locale is set automatically based on the home assistant locale. 

A list of the current supported locales is in the documents

**Comment by neffmartin** [comment]
yes i know, the request was to just add a sentence to the manual how to configure it manually
